modules/ROOT/pages/authentication-authorization/limitations.adoc

```diff
@@ -281,9 +281,9 @@ will only return label `:A`, because that is the only label for which traversal
 [[access-control-limitations-db-operations]]
 == Security and count store operations
 
-The rules of a security model may impact some of the database operations.
-This means extra security checks are necessary to incur additional data accesses, especially in the case of count store operations.
-These are, however, usually very fast lookups and the difference might be noticeable.
+The rules of a security model may impact some database operations.
```

Comment:
The problem is that is it not only for count store operations, even if it is a good example because it is very clear that it will be slower in that case and it is easy to explain why.

Reply:
Then, we need to change the heading and write in general and mention the count store operations just as an example.

Reply:
And perhaps also mention the other impacted operations.

Reply:
The second example, under the sub-header `Property-based access control limitations` is an other example which has nothing to do with count store but more a general example why it will impact performance.

Reply:
> Then, we need to change the heading and write in general and mention the count store operations just as an example.

What do you think about header: "Security rules impact on performance" ? 
or something similar